core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -83,7 +83,7 @@ private[spark] class Executor(
   extends Logging {
 
   logInfo(log"Starting executor ID ${LogMDC(LogKeys.EXECUTOR_ID, executorId)}" +
-    log" on host ${LogMDC(HOST_NAME, executorHostname)}")
+    log" on host ${LogMDC(HOST, executorHostname)}")
```

Comment:
Let's unify it as `HOST`, the general rule is
HOST: Only `host name`, not including `port`, eg: host_name
Port: Just the `port`, eg: 10240
HOST_PORT: includes both host name and port, eg: `host_name:10240`

Reply:
https://github.com/apache/spark/blob/e972daec30eb34e9352f37e9ac74db77a895d679/resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnAllocator.scala#L765